pyproject.toml

```diff
@@ -11,7 +11,7 @@ authors = [
 dependencies = [
     "langchain-core>=0.1.1, <1.0.0",
     "langchain-community>=0.0.18, <1.0.0",
-    "google-cloud-firestore>=2.14.0, <3.0.0",
+    "google-cloud-firestore@git+https://github.com/pl04351820/python-firestore.git@vector",
```

Comment:
Can you please elaborate more on this change? -- I assume it's temporal until the Vector is in Public Review?

Reply:
Can you set the minimum to be 2.16.0 (https://github.com/googleapis/python-firestore/pull/876)